### PR TITLE
COMMERCE-12598 - Automation for COMMERCE-12012 part2

### PR DESCRIPTION
**Pull request by jaydawu**
Test plan: https://liferay.atlassian.net/browse/COMMERCE-12477

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 1]
```diff
@@ -2747,6 +2747,102 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12598. As an admin, I can verify the order items are split on the order admin when the 'Show Separate Order Items' toggle is enabled"
+	@priority = 5
+	test CanVerifyOrderItemsAreSplitOnTheOrderAdmin {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2752]:
```suggestion
	test CanAssertOrderItemsAreSplitInOrdersAdmin {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 2]
```diff
@@ -2747,6 +2747,102 @@ definition {
...
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			for (var count : list "1,2") {
+				Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+				Click(
+					key_productName = "U-Joint",
+					locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+			}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2799]: Can you add to the cart with different quantities? This way you can assert both rows later on during the test. Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase [hunk 3]
```diff
@@ -913,6 +913,125 @@ definition {
...
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			for (var count : list "1,2") {
+				Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+				Click(
+					key_productName = "U-Joint",
+					locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+			}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase:961]: Can you add to the cart with different quantities? This way you can assert both rows later on during the test. Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase [hunk 4]
```diff
@@ -913,6 +913,125 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12600. As a buyer, I can verify the order items are split on the placed order when the 'Show Separate Order Items' toggle is enabled"
+	@priority = 5
+	test CanVerifyOrderItemsAreSplitOnThePlacedOrder {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase:918]:
```suggestion
	test CanAssertOrderItemsAreSplitInPlacedOrders {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 5]
```diff
@@ -1829,6 +1829,143 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12599. As an admin, I can verify the order items are split on the order shipments section with correct quantities when the 'Show Separate Order Items' toggle is enabled"
+	@priority = 5
+	test CanVerifyOrderItemsAreSplitOnOrderShipmentsWithCorrectQuantity {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:1834]:
```suggestion
	test CanAssertOrderItemsAreSplitInShipments {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 6]
```diff
@@ -2747,6 +2747,102 @@ definition {
...
+				locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+				value1 = 2);
+
+			Pagination.viewResults(results = "Showing 1 to 2 of 2 entries.");
+
+			CommerceOrders.viewOrderSummary(
+				deliveryValue = "$ 15.00",
+				subtotalValue = "$ 48.00",
+				totalValue = "$ 63.00");
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2843]: Can you use a user with the Operations Manager role to do the assertions? This applies also to the shipments test. Thanks!

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@jaydawu can you move these tests all in a new .testcase file for the order items split?
If you could also move the tests already merged to this new file that would be great! Thanks a lot!

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Review by lorenzolapescara — APPROVED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**